esmvaltool/recipes/recipe_autoassess_radiation_rms_cfMon_all.yml

```
@@ -61,7 +61,7 @@ diagnostics:
         derive: true
         force_derivation: false
     additional_datasets:
-      - {dataset: ISCCP,  project: obs4mips,  level: L3,  version: V1.0,  start_year: 1985,  end_year: 1988, tier: 1}
+      - {dataset: ISCCP,  project: obs4mips, mip: cfMon, frequency: mon, level: L3,  version: V1.0,  start_year: 1985,  end_year: 1988, tier: 1}
```

```suggestion
      - {dataset: ISCCP,  project: obs4mips, level: L3,  version: V1.0,  start_year: 1985,  end_year: 1988, tier: 1}
```
I think this change is not needed?